bin/crew

```diff
@@ -124,6 +124,25 @@ String.use_color = args['--color'] || !args['--no-color']
 @verbose           = @opt_verbose ? 'v' : ''
 @short_verbose     = @opt_verbose ? '-v' : ''
 
+class ExitMessage
+  attr_accessor :msg
+
+  @obj_all = []
+
+  def initialize(msg)
+    @msg = msg
+  end
+
+  class << self
+    attr_reader :obj_all
+
+    def new(msg)
+      obj_all << super
+      obj_all.last
```

Comment:
Do you mean `obj_all << msg`? (the message string instead of the `ExitMessage` class)

This line can be removed as the line above will return the message itself.

Reply:
(actually, it is better to use `.add` as the method name to prevent confusing)

Reply:
I'm modifying this:

https://stackoverflow.com/a/53877094/21935208